README.md

```diff
@@ -24,11 +24,10 @@ graph TD
     - REDMINE_URL - URL to target Redmine from where tickets will be dumped
     - REDMINE_KEY - API key allowing to access REDMINE_URL
     - REDMINE_QUERY_RATIO - Max request per minute to the redmine API
-    - MIRROR_SQLITE_DB - Sqlite file DB where tickets will be dumped
 3. execute :
 
 ```bash
-podman run  -ti --rm -v <your_redmine_config_path>:/pms/redmine/config.py pms "./dumper.py --project <project_name>"
+podman run  -ti --rm -v <your_redmine_config_path>:/pms/redmine/config.py pms "./dumper.py --project <project_name> --database <sqlite_file> "
```

Comment:
it would be nice to comment the `<project_name>` as it was not obvious to me in the first look. the whole command is very much about the redmine which is not mentioned and adding bugzilla might cause some confusion. Just try to think ahead but not big deal atm.

Reply:
Will the database be used for Bugzilla? If the argument is just `database` I would assume it will be

Reply:
@kalikiana  We could use the database for bugzilla. The objective is to have an standarized DB.  Also dumper.py is prepared for other sources: https://github.com/ilausuch/project_management_statistics/blob/902b37373a4d942aae112f6968efb6c879d9c02d/dumper.py#L10

However only progress is implemented right now. @b10n1k Maybe we could merge this and do some specific changes for bugzilla, for instance, where the project is not required. Could you help me on that?